extensions/wikia/UserLogin/js/UserLoginModal.js

```diff
@@ -82,6 +82,7 @@
 
 					UserLoginModal.loginAjaxForm = new window.UserLoginAjaxForm($loginModal, {
 						ajaxLogin: true,
+						modal: loginModal,
```

Comment:
Yeah it's being used for the reset password functionality when it's done inside a modal.